sdk/core/src/request_options/next_marker.rs

```diff
@@ -39,6 +39,28 @@ impl NextMarker {
             .filter(|h| !h.is_empty())
             .map(|h| NextMarker::new(h.to_owned())))
     }
+
+    //#[cfg(feature = "table")]
```

Comment:
What's with the comments?

Reply:
Why are these in core?

Reply:
They are in core because `next_marker` is used everywhere for paginated responses. Trouble is, table API use a different header then the rest (`x-ms-continuation-NextTableName`). Do you think it would be better to create a table specific `next_marker` type? 

As for the comments, I wanted to gate the table functions to the `table` feature (they do not make sense otherwise) but of course it didn't work because the table feature is in the storage crate 🤷‍♂️ . 

I will try to create a storage table specific `next_marker`, this will make these problems disappear.

Reply:
Done! It's much clearer now. 👍